test/simple/test-timers-active-handles.js

```diff
@@ -0,0 +1,36 @@
+var cp = require('child_process');
+var assert = require('assert');
```

Comment:
use `const` wherever you assign the value once and never reassign anything to the same.